sds_data_manager/lambda_code/SDSCode/database/models.py

```diff
@@ -0,0 +1,33 @@
+"""Main file to store schema definition"""
+from sqlalchemy import (
+    Boolean,
+    Column,
+    DateTime,
+    Integer,
+)
+from sqlalchemy.orm import DeclarativeBase
+
+
+class Base(DeclarativeBase):
+    pass
+
```

Comment:
```suggestion
Base = sa.orm.declarative_base()
```
If you want to do this in one less line!  In my experience, I don't think we should ever need to define things in the `Base` class(?)

Reply:
That's what I saw in others but I found this way to be more consistence with object oriented.